ipaserver/rpcserver.py

```diff
@@ -368,12 +369,15 @@ def wsgi_execute(self, environ):
         args = ()
         options = {}
         command = None
+        time_start = None
+        time_end = None
```

Comment:
Do these need to be defined here? I guess it could be good practice, but I don't see a way that either could be used without calling time.perf_counter() first.